packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -447,29 +460,27 @@ export default {
                 await RegistrationServiceApi.createAccount(this.createAccountUrl, registrationData, this.tenant);
                 this.$emit(EventNames.CreateAccountSuccess);
             } catch (error) {
-                let thrownErrors = error;
-                if (error && error.response && error.response.data && error.response.data.errors) {
-                    thrownErrors = error.response.data.errors;
-                }
-                let shouldEmitCreateAccountFailure = true;
+                if (error.response && error.response.status) {
+                    if (error.response.status === 409) {
```

Comment:
Minor thing, but since we're using `error.response.status` several times, maybe it's worth extracting it into a `const`?